SELF_HOSTED_INSTRUCTIONS.md

```diff
@@ -0,0 +1,48 @@
+## First one time set up
```

Comment:
Maybe hyphen one time? "First one-time setup" 

Also, before the fist line, I would add a line saying something like:  "This example shows the simplest approach at having a working Uwazi.  You can further finetune paths by using individual configurations for each folder, for more info please read `xxx`"